libs/application/core/src/types/Application.ts

```diff
@@ -45,3 +45,5 @@ export interface Application {
   progress?: number
   status: ApplicationStatus
 }
+
+export type Application = Omit<ApplicationWithAttachments, 'attachments'>
```

Comment:
maybe the other way around and we can stick with interfaces

```
interface Application {
...
}
```

```
interface ApplicationWithAttachments extends Application {
attachments...
}
```

Reply:
Yes sure!